### DISCUSSION TITLE
How to write failureCondition when kubectl resources is form as a list

### DISCUSSION THREAD

**Discussion by caydenwei** [argoproj/argo-workflows#7434] [Q&A] [answered]
I try to use argo workflow to generate Kubernetes resources, I saw the log,  argo uses
```
kubectl get xxx -w -o json -n xxx
```
to get the resources info, and the resource gives the following. I want to match one on the "type": "Failed" in conditions as evaluation of failureCondition,  how should I write the syntex?

```
...
"status": {
        "completionTime": "2021-12-17T03:12:03Z",
        "conditions": [
            {
                ...
                "type": "Created"
            },
            {
               ...
                "type": "Failed"
            },
            {
                ...
                "type": "Running"
            }
        ],
        "replicaStatuses": {
            "Launcher": {},
            "Worker": {}
        },
        "startTime": "2021-12-17T03:04:38Z"
    }
...
```

**Comment by terrytangyuan** [comment, marked as answer]
You can use something like: status.replicaStatuses.Worker.succeeded == 3

Looks like you are using [Kubeflow MPI Operator](https://github.com/kubeflow/mpi-operator)? If so, this should be sufficient for most of your use cases.
